pairtools/cli/flip.py

```diff
@@ -101,7 +101,7 @@ def flip_py(pairs_path, chroms_path, output, **kwargs):
     ]
 
     for line in body_stream:
-        cols = line.rstrip().split(pairsam_format.PAIRSAM_SEP)
+        cols = line.rstrip('\n').split(pairsam_format.PAIRSAM_SEP)
```

Comment:
what's another alternative here?

Reply:
Why not just keep it like it was before?

Reply:
We never know what might be the content of the last column of the pairs file. It might be empty, and then stripping will erase this information (latest `pairtools split` problem). 
In general, operations should do only the things that are their primary purpose. The purpose of this stripping was to get rid of new line characters and not to strip any whitespace.

Reply:
OK!